php/providers/AutocompleteProvider.php

```diff
@@ -55,23 +55,28 @@ public function execute($args = array())
                 return $this->getClassMetadata($values['declaringClass']);
             }
         } elseif (ucfirst($returnValue) === $returnValue) {
-            $parser = new FileParser($classMap[$class]);
+            // At this point, this could either be a class name relative to the current namespace or a full class
+            // name without a leading slash. For example, Foo\Bar could also be relative (e.g. My\Foo\Bar), in which
+            // case its absolute path is determined by the namespace and use statements of the file containing it.
+            $className = $returnValue;
 
-            $found = false;
-            $className = $parser->getCompleteNamespace($returnValue, $found);
+            if (!empty($className) && $returnValue[0] !== "\\" && isset($classMap[$values['declaringClass']])) {
+                $parser = new FileParser($classMap[$values['declaringClass']]);
```

Comment:
I'm afraid I'm a bit confused: do you mean that the brace `{` is missing behind the if-statement? The if-statement itself is more than one line and the brace is at the end of line 63. Are you perhaps referring to the use of the parantheses `(` and `)` for the conditions?